canvas_modules/common-canvas/__tests__/common-properties/components/title-editor-test.js

```diff
@@ -270,13 +297,13 @@ describe("title-editor renders correctly", () => {
 				showHeading
 			/>
 		);
-		expect(wrapper.find(".properties-title-heading")).to.have.length(1);
-		expect(wrapper.find(".properties-title-editor.properties-title-with-heading")).to.have.length(1);
-		expect(wrapper.find(".properties-title-heading-label")).to.have.length(1);
-		expect(wrapper.find("InlineSVG.properties-title-heading-icon")).to.have.length(1);
+		const { container } = wrapper;
+		expect(container.getElementsByClassName("properties-title-heading")).to.have.length(1);
+		expect(container.getElementsByClassName("properties-title-editor properties-title-with-heading")).to.have.length(1);
+		expect(container.getElementsByClassName("properties-title-heading-label")).to.have.length(1);
 	});
```

Comment:
here there is no check for `expect(wrapper.find("InlineSVG.properties-title-heading-icon")).to.have.length(1);`

Reply:
I removed it because RTL and querySelector can't see InlineSVG.properties-title-heading-icon. When I use the debugger to show the HTML, the inlinesvg and the properties-title-heading-icon class don't appear 
<img width="1131" alt="image" src="https://github.com/user-attachments/assets/5761ba3a-aad0-476a-8125-9c3518ed4588">